### ISSUE TITLE
System requirements not aligned to Node.js release schedule

### ISSUE THREAD

**Issue by MikeMcC399** [cypress-io/cypress-documentation#5396]
### Subject

Getting Started > Installing Cypress > Node.js

### Description

[Getting Started > Installing Cypress > Node.js](https://docs.cypress.io/guides/getting-started/installing-cypress#Nodejs) includes versions of Node.js which have passed into end-of-life status. This is no longer in line with the [Node.js release schedule](https://github.com/nodejs/release#release-schedule):

- Node.js `14` passed into end-of-life status on Apr 30, 2023
- Node.js `19` passed into end-of-life status on Jun 1, 2023
- Node.js `20` was released on Apr 18, 2023

(in addition

- Node.js `16` is planned to transition into end-of-life status on Sep 11, 2023)

![image](https://github.com/cypress-io/cypress-documentation/assets/66998419/c3b45dc6-2bca-49a7-80da-231cdf90014a)

**Comment by jaffrepaul** [comment]
Good catch @MikeMcC399. I checked with the team and bumping the supported node versions in v13 didn't make the cut based on available resources. But they'd be happy to review a PR if you wanted to give making that change a shot. To my knowledge it would require:

- [editing the versions here](https://github.com/cypress-io/cypress/blob/d0fbd29805e5e8bb0778936a25154fd516fce935/cli/package.json#L126) (as you noted)
- [creating a new docker image](https://github.com/cypress-io/cypress-docker-images)
- [adjusting tests](https://github.com/cypress-io/cypress/blob/develop/system-tests/test-binary/node_versions_spec.ts) accordingly

**Comment by MikeMcC399** [comment]
Hi @jaffrepaul and thanks for checking with the team!

- I have submitted PR #5401 to make changes to the documentation [Getting Started > Installing Cypress > Node.js](https://docs.cypress.io/guides/getting-started/installing-cypress#Nodejs) so that it conforms to the [Node.js release schedule](https://github.com/nodejs/release#release-schedule). This will help guide users so that they know which Node.js versions they should be using. The updated documentation list is stricter than Cypress' current `engine` definition and it is OK for the `engine` enforcement to lag behind. That will not break any user workflows. The PR is sufficient to close this issue.

- At this time I don't have enough experience to tackle the Cypress core and Cypress Docker image PRs that you suggested in order to update all the related bits and pieces, so I would not feel comfortable attempting this, especially as it involves a breaking change.

- This topic should be revisited when  Node.js `16` enters end-of-life (planned for on Sep 11, 2023). Currently Cypress `12.x` is locked to Node.js `16.16.0` (see [.node-version](https://github.com/cypress-io/cypress/blob/develop/.node-version)). There is also nothing so far in the [release/13.0.0](https://github.com/cypress-io/cypress/blob/release/13.0.0) branch to change this. The [cypress-io/github-action](https://github.com/cypress-io/github-action) Cypress JavaScript Action is locked to `node16` (see https://github.com/cypress-io/github-action/issues/968) until GitHub provides support for `node20`, so I would expect there to be a phase when Cypress needs to continue working with Node.js `16` even after it has passed its end-of-life date.
